src/mdoc/model/document.ts

```diff
@@ -131,9 +142,20 @@ export class Document {
         if (typeof value !== 'string') {
           throw new Error(`Invalid type for 'birth_date'. Expected 'string', received '${typeof value}'`)
         }
+
         const ageInYears = getAgeInYears(value)
-        addAttribute('age_over_21', ageInYears >= 21)
-        addAttribute(`age_over_${Math.floor(ageInYears)}`, true)
+        const addAgePredicate = (predicate: number) => addAttribute(`age_over_${predicate}`, ageInYears >= predicate)
+
+        // Age predicates.
+        // Values defined in: https://bmi.usercontent.opencode.de/eudi-wallet/eidas-2.0-architekturkonzept/functions/00-pid-issuance-and-presentation/#pid-contents
+        // ISO/IEC 18013-5: 7.2.5 does not define which ages should be included.
+
+        addAgePredicate(12)
```

Comment:
Should this be defined by this library? Why not just leave it to the user / implementer of this library?

Reply:
Yeah that can also work, but maybe @auer-martin could pick that up. I am not really aware of the correct API changes that need to be made for this.